enterprise/service/license.go

```diff
@@ -62,8 +62,31 @@ func (s *LicenseService) StoreLicense(ctx context.Context, patch *enterpriseAPI.
 	return err
 }
 
-// LoadLicense will load license from file and validate it.
-func (s *LicenseService) LoadLicense(ctx context.Context) (*enterpriseAPI.License, error) {
+// LoadSubscription will load subscription.
+func (s *LicenseService) LoadSubscription(ctx context.Context) (enterpriseAPI.Subscription, error) {
```

Comment:
Just let this func return the subscription? (we will always ignore the error)

Reply:
Good idea, also moving cache and a few other enterprise related code to license service.